mattermost/files/mattermost.service

```diff
@@ -1,8 +1,6 @@
 [Unit]
 Description=Mattermost
 After=network.target
-After=postgresql.service
```

Comment:
We remove `postgresql` hardcoded dependency from mattermost-server packaging, since it is in the user's repsonsibility/choice to select his favorite db service (postgresql - mysql) 
Keeping it non opinionated. 

When shipping with omnibus-mattermost, we define this as postgresql (Highly opinionated)

Reply:
ok so for omnibus will stay as is got it.

Reply:
That's correct